courses/models.py

```diff
@@ -287,3 +288,30 @@ def satisfying_courses(self):
         return Course.objects.all()\
             .exclude(id__in=self.overrides.all())\
             .filter(Q(department__in=self.departments.all(), semester=self.semester) | Q(id__in=self.courses.all()))
+
+
+"""
+3rd-Party API
+"""
+
+PCA_REGISTRATION = 'PCA_REGISTRATION'
+
+
+class APIPrivelege(models.Model):
```

Comment:
Typo here. APIPrivelege -> APIPrivilege. This would need to be changed everywhere.